config/environments/development.rb

```diff
@@ -18,9 +18,9 @@
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
+    config.action_controller.enable_fragment_cache_logging = true
 
     config.cache_store = :mem_cache_store,
-                         'localhost:11211',
```

Comment:
Is this default value now?

Reply:
seems it was default value for whole time, anyway is still inc urrent version -> https://github.com/rails/rails/blob/v6.0.0/activesupport/lib/active_support/cache/mem_cache_store.rb#L70